New translates added

darkjoya opened the pull request:
New translated strings. I cant find any [credits] section to add my name but its ok.

Teknoist commented:
https://accounts.klei.com/community-translations

cixtor commented:
Hello @darkjoya thank you very much for your contribution.

I can add your name to the credits section of the “Community Translations” page, is it okay if I use your username `darkjoya` or do you prefer a different name?

By the way, there seems to be a conflict between the current content of the Turkish localization file and the changes you are proposing, GitHub will not let us merge your changes until the conflicts are fixed, can you please review your translations one more time?

Best regards.

darkjoya commented:
There is 2 wrong translations so i changed them. What am i supposed to do with them? I'm writing down them and 

I changed the translate of "#. i18n: keyword: Email" because previous translator use "E-Mail" but its not a true translate. Word "E-Mail" has a Turkish equivalent so i changed with the true words (EPosta).
Also in "#. i18n: keyword: MergeAccountSuccessTitle" the other translation is an inverted sentence and its not true so i fixed this.
